examples/common/python/utility/signature.py

```diff
@@ -181,11 +181,14 @@ def generate_signature(self, hash, private_key):
             - hash is the combined array of all hashes calculated on the message
             - private_key is Client private key
         """
-
-        self.private_key = private_key
-        self.public_key =  self.private_key.GetPublicKey().Serialize()
-        signature_result =  self.private_key.SignMessage(hash)
-        signature_base64  =  crypto.byte_array_to_base64(signature_result)
+        try:
+            self.private_key = private_key
+            self.public_key =  self.private_key.GetPublicKey().Serialize()
+            signature_result =  self.private_key.SignMessage(hash)
+            signature_base64  =  crypto.byte_array_to_base64(signature_result)
+        except:
+            logger.error("Exception occured during signature generation")
+            return SignatureStatus.FAILED
```

Comment:
Better to return consistent types on success and failure. One option is to return a tuple with (signature, status). Also please update function comments accordingly

Reply:
Done